example/metadata/lib/src/generated/metadata.pb.dart

```diff
@@ -0,0 +1,109 @@
+///
```

Comment:
It looks a little bit funky with the two lines of empty doc comments in the header.